families/track_and_trade/run-tnt.sh

```diff
@@ -0,0 +1,24 @@
+# Copyright 2017 Intel Corporation
```

Comment:
Suggest removing this script and (probably in another PR) adding a client docker container with the client installed.  This will remove the requirement to install npm locally on the host.  It will also allow running docker compose in the foreground, which is useful for watching logs.

Reply:
Implied here is doing something like "docker exec -it tnt-client bash" to get into the container after it's been run using docker compose.